pennylane_lightning/src/algorithms/AdjointDiff.cpp

```diff
@@ -18,6 +18,9 @@
 template class Pennylane::Algorithms::AdjointJacobian<float>;
 template class Pennylane::Algorithms::AdjointJacobian<double>;
 
+template class Pennylane::Algorithms::VectorJacobianProduct<float>;
+template class Pennylane::Algorithms::VectorJacobianProduct<double>;
```

Comment:
Do you think this should be included in the Adjoint file, or separated into its own file in the algorithms directory?

Reply:
it's definitely better to be added into its own file namely `JacobianProd.cpp` in the same directory.